src/content/nl/2020/cms.md

```diff
@@ -0,0 +1,527 @@
+
+Welk CMS er ook wordt gebruikt, het wordt gebruikt omdat het een probleem oplost voor de gebruiker of organisatie. Het valt buiten ons bereik om te onderzoeken waarom elk CMS wordt gekozen, maar later onderzoeken we waarom het meest populaire CMS, WordPress, onevenredig wordt gekozen.
+
+## CMS-adoptie
+
+Onze analyse tijdens dit werk kijkt naar desktop- en mobiele websites. De overgrote meerderheid van de URL's die we hebben bekeken, bevindt zich in beide datasets, maar sommige URL's zijn alleen toegankelijk voor desktop- of mobiele apparaten. Hierdoor kunnen kleine verschillen in de data ontstaan en kijken we dus apart naar desktop- en mobiele resultaten.
+
+Meer dan 42% van de webpagina's wordt aangedreven door een CMS-platform, een stijging van meer dan 5% ten opzichte van 2019. Dit komt neer op 42,18% op desktop, tegen 40,01% in 2019, en 42,27% op mobiel, tegen 39,61% in 2019.
+
+{{ figure_markup(
+  image="cms-adoption.png",
+  caption="CMS-acceptatietrend.",
+  description="Staafdiagram met de toename van het niveau van CMS-acceptatie in 2019 en 2020. Mobiel is gestegen van 39,61% naar 42,47%. Desktop is gestegen van 40,01% naar 42,18.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=2061880890&format=interactive",
+  sheets_gid="1638210080",
+  sql_file="cms_adoption_compared_to_2019.sql"
+  )
+}}
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Jaar</th>
+        <th>Desktop</th>
+        <th>Mobiel</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>2019</td>
+        <td class="numeric">39,61%</td>
+        <td class="numeric">40,01%</td>
+      </tr>
+      <tr>
+        <td>2020</td>
+        <td class="numeric">42,27%</td>
+        <td class="numeric">42,18%</td>
+      </tr>
+      <tr>
+        <td>% Verandering</td>
+        <td class="numeric">6,71%</td>
+        <td class="numeric">5,43%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="CMS-adoptiestatistieken.", sheets_gid="1638210080", sql_file="cms_adoption_compared_to_2019.sql") }}</figcaption>
+</figure>
+
+De toename van desktopwebpagina's aangedreven door een CMS-platform is 5,43% ten opzichte van vorig jaar. Op mobiel is deze stijging ongeveer een kwart hoger, namelijk 6,71%.
+
+Net als bij [vorig jaar](../2019/cms#cms-adoption), zien we andere resultaten van andere datasets voor het bijhouden van marktaandeel van CMS-platforms, zoals [W3Techs](https://w3techs.com/technologies/history_overview/content_management). W3Techs meldt op het moment van schrijven dat 60,6% van de webpagina's wordt gemaakt door CMS'en, tegen 56,4% een jaar geleden. Dit is een stijging van 6,4%, wat in grote lijnen overeenkomt met onze bevindingen.
```

Comment:
That `hreflang` functionality does mean you have to convert markdown links to HTML links if you want to use them which is a bit of extra effort 😢 (we probably should change all the English ones to make this easier for Translators):

```suggestion
Net als bij [vorig jaar](../2019/cms#cms-adoption), zien we andere resultaten van andere datasets voor het bijhouden van marktaandeel van CMS-platforms, zoals <a hreflang="en" href="https://w3techs.com/technologies/history_overview/content_management">W3Techs</a>. W3Techs meldt op het moment van schrijven dat 60,6% van de webpagina's wordt gemaakt door CMS'en, tegen 56,4% een jaar geleden. Dit is een stijging van 6,4%, wat in grote lijnen overeenkomt met onze bevindingen.
```